stubs/html5lib/METADATA.toml

```
@@ -1,5 +1,7 @@
 version = "1.1.*"
 upstream_repository = "https://github.com/html5lib/html5lib-python"
 
+requires = ["six"]
```

If you want to use types for six, you need to add types-six since the types are distributed separately:
```suggestion
requires = ["types-six"]
```